lib/Db/PimcoreExtensionsTrait.php

```diff
@@ -611,4 +611,28 @@ public function setAutoQuoteIdentifiers($autoQuoteIdentifiers)
     {
         $this->autoQuoteIdentifiers = $autoQuoteIdentifiers;
     }
+
+    /**
+     * @param string $table
+     * @param string $idColumn
+     * @param string $where
+     */
+    public function selectAndDeleteWhere($table, $idColumn = 'id', $where = '')
+    {
+        $sql = 'SELECT ' . $idColumn . '  FROM ' . $table;
```

Comment:
i think $idColumn should be quoted here?

Reply:
👍 good catch!  `quoteIdentifier()`
@weisswurstkanone can you please update the PR! Thanks!

Reply:
thx!